extensions/wikia/Forum/RelatedForumDiscussionController.class.php

```diff
@@ -101,4 +109,8 @@ public static function getData( $articleId ) {
 			return $messages;
 		});
 	}
+
+	private static function logError( $message, array $param = [] ) {
+			WikiaLogger::instance()->error( 'RelatedForumDiscussionController: ' . $message, $param );
```

Comment:
Sorry, need to be more specific :) The indentation seems larger than what it should be